backend/src/legacy/EventController.test.ts

```diff
@@ -11,10 +11,13 @@ beforeAll(async () => {
 
   testApp = app;
   testConnection = connection;
+  console.log(testApp, testConnection);
 });
 
 afterAll(async () => {
-  await testConnection.close();
+  if (testConnection != null) {
+    await testConnection.close();
+  }
```

Comment:
For this file, it's OK to just not put any changes. Main already has this test under a legacy folder. I believe what should happen is when merging the branch, it will only merge changed files, so leaving this out is alright.